src/raster/r.damflood/r.damflood.html

```diff
@@ -1,140 +1,197 @@
+expressed in meters (m). <br>
 
 
 
-Optionally the user may modify the initial timestep used for the numerical solution of the SWE (<em>default value = 0.01 s</em>), nevertheless the timestep  [],
-and choose a specific failure tipe corresponding to different computational method for the initial velocity estimation. 
+Optionally the user may modify the initial timestep used for the numerical
+solution of the SWE (<em>default value = 0.01 s</em>), nevertheless the
+timestep [<b>timestep</b>],
+and choose a specific failure type corresponding to the different computational
+methods for the initial velocity estimation.
 <br><br>
 
 
-<br>
-
-<br>
-<br>
-<em><b><!--font size = "5"-->Notes</b></em><!--/font-->
 
-<br>
-<br>
+<h2>NOTES</h2>
 <em>(GRASS ANSI C command)</em>
 
 
-<h2>AUTHORS</h2>
-Roberto Marzocchi (<a href="mailto:roberto.marzocchi@gter.it">e-mail</a>) and Massimiliano Cannata (<a href="mailto:massimiliano.cannata@supsi.ch">e-mail</a>).
-The GRASS tool was developed by Institute of earth science (IST), 
-University of applied science of Italian Switzerland (SUPSI), Lugano - Division of geomatics <a href="http://istgeo.ist.supsi.ch/site/" target="_blank">web-page</a><br>
-Actually the debug is assured by:<br>
- - <a href="http://www.gter.it/?q=en"  target="_blank"> Gter srl</a> (Genoa, Italy) <br>
- - <a href="http://www.ist.supsi.ch/" target="_blank">IST -SUPSI</a> (Lugano, Switzerland)<br>
-
-   
-
-The numerical model, originally developed by the National Center for Computational Hydroscience and Engineering of the University of Mississippi, 
-has been reformulated and modified by the authors introducing important new features to consider the numerical stability and the type of dam failure, 
-and currently is written in ANSI C programming language within GRASS.<br><br>
-
+<h2>REFERENCES</h2>
+[1] Cannata M. & Marzocchi R. (2012). Two-dimensional dam break flooding simulation: a GIS embedded approach. - Natural Hazards 61(3):1143-1159 <br>
+[2] <a href="http://gfoss2009.crs4.it/en/system/files/marzocchi_cannata_licensed.pdf" 
+target="_blank">Pdf</a> presentation of the work at the 
+"X Meeting degli Utenti Italiani di GRASS - GFOSS" (It) 
+<a href="http://gfoss2009.crs4.it/en/node/61" target="_blank">web-page</a><br>
+[3] Pdf presentation of the work at the FOSS4G 2009 (En) - 
+<a href="http://2009.foss4g.org/researchpapers/#researchpaper_10" 
+target="_blank">web-page</a><br>
+[4] Pdf presentation of the work at the Geoitalia 2011 conference (En) - 
+<a href="https://dl.dropbox.com/u/3019930/marzocchi_cannata_geoitalia2011.pdf"
+target="_blank">document</a><br>
 
 
 <h2>SEE ALSO</h2>
 <em><a href="https://grass.osgeo.org/grass-stable/manuals/r.lake.html">r.lake</a></em>,
 <em><a href="https://grass.osgeo.org/grass-stable/manuals/r.reclass.html">r.reclass</a></em>,
-<em><a href="https://grass.osgeo.org/grass-stable/manuals/d.rast.arrow.html">d.rast.arrow</a></em>,
-<em><a href="r.inund.fluv.html">r.inund.fluv</a></em>.
```

Comment:
r.inund.fluv was a grass6 fortran addon, not existing in grass 7 or grass 8. https://github.com/OSGeo/grass-addons/blob/a7f4449bd06f9ba26ec88d73468d2ccf334eb455/src/raster/r.inund.fluv/README.txt